src/pages/Admin/Venue/VenueDetailsForm.tsx

```diff
@@ -0,0 +1,297 @@
+import React, {
```

Comment:
Renamed `DetailsForm` -> `VenueDetailsForm` to distinguish it from the other component with the same name